Makefile

```diff
@@ -37,5 +37,53 @@ image: generate
     		--tag rss3-network/node:$(VERSION) \
     		.
 
-run: generate
-	ENVIRONMENT=development go run ./cmd
+# Run a worker service locally to index and structure data from a decentralized source
+# into the RSS3 Protocol format. Use `make worker WORKER_ID=<worker-id>`.
+# Example: `make worker WORKER_ID=ethereum-uniswap` runs one worker to index and transform data from the Uniswap dApp source.
+#
+# To use this command:
+# - Set the decentralized network endpoint (to which the decentralized source belongs) in the 'endpoint' section of deploy/config.yaml.
+# - Configure the decentralized data source component (e.g., a specific blockchain or its dApp) with the necessary parameters
+#   in the 'component' section of deploy/config.yaml.
+# - Use the worker ID of the component as WORKER_ID in the make command.
+worker: generate
+	@if [ -z "$(WORKER_ID)" ]; then \
+		echo "WORKER_ID is not set. Usage: make worker WORKER_ID=<worker-id>"; \
+		exit 1; \
+	fi
+	ENVIRONMENT=development go run ./cmd \
+			--module=worker \
+			--worker.id=$(WORKER_ID)
+
+
+# Run the core service of the RSS3 Node locally, providing indexed data conforming to the RSS3 protocol to end users.
+# The node serves API requests on port 80 by default. Use `make core`.
+
+# Note: When routing API requests to check indexed data, include the routing group name in some paths.
+# For example, API services within 'internal/node/component/decentralized' may require adding '/decentralized' to
```

Comment:
I think the code is self-explanatory of this part, so there's no need to mention it in a Makefile comment.

It may be more appropriate to include this information in a detailed quick start guide or a developer documentation if we have later.

Reply:
Thanks for your feedback. I have added commands for setting up the resources to run node services. Also I've included a developer guide to assist users in setting up the development environment and using the commands.

Reply:
Thanks for your feedback. I have added commands for setting up the resources to run node services. Also I've included a developer guide to assist users in setting up the development environment and using the commands.